Tests/Remora.Discord.Gateway.Tests/Constants.cs

```diff
@@ -37,7 +37,7 @@ public static class Constants
     public static IUser BotUser { get; } = new User
     (
         DiscordSnowflake.New(0),
-        "mock-bot",
```

Comment:
Any particular reason for this change?

Reply:
Compiler error, though it seems me trying to reflect upstream hasn't gone well

Reply:
However it's worth noting that hyphens aren't ~technically~ supported in pomelo